README.md

```diff
@@ -1 +1,102 @@
+## Using Git in VS Code
+In this guide I will tell you how you, yes you could use git through VS code making your life significantly easier than before. Am I saying you will never have to use console technically as far as VS code is concerned yes with pushing and pulling. You do not have to use console except when you would need to roll back after pushing changes, let’s get started.
 
+### Step 1
+Open VS Code
+
+### Step 2 
+Locate your local repo or if you have already opened the repo before with or without using a Dev Container you can find it in a list of 5 items below the section labeled recently opened. If you re opened a Dev Container repo you opened before skip to Step 5.
```

Comment:
Run this through a grammar tool to fix issues. Again, prefer much shorter sentences.